pkg/api-server/testdata/resources/inspect/dataplanes/_rules/dataplane_kind_meshtimeout_section_name.golden.json

```diff
@@ -0,0 +1,83 @@
+{
+ "httpMatches": [],
+ "resource": {
+  "labels": {
+   "app": "demo-app"
+  },
+  "mesh": "mesh-1",
+  "name": "dp-1",
+  "type": "Dataplane"
+ },
+ "rules": [
+  {
+   "fromRules": [
+    {
+     "inbound": {
+      "port": 8080,
+      "tags": {
+       "kuma.io/service": "foo"
+      }
+     },
+     "rules": [
+      {
+       "conf": {
+        "connectionTimeout": "7s",
+        "idleTimeout": "7s",
+        "http": {
+         "requestTimeout": "7s"
+        }
+       },
+       "matchers": [],
+       "origin": [
+        {
+         "labels": {},
+         "mesh": "mesh-1",
+         "name": "select-whole-dpp",
+         "type": "MeshTimeout"
+        }
+       ]
+      }
+     ]
+    },
+    {
+     "inbound": {
+      "port": 9090,
```

Comment:
we probably should add a `name` field here, it'll be helpful for GUI

Reply:
added